File: Selnium script for testing/Script with Python/test_scenario_47.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_47():
    """
    Dummy test run for scenario 47
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_47")

    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_555")
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.ID, "checkout_btn").click()
    driver.find_element(By.NAME, "password_field").send_keys("test_data_541")
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.ID, "user_name").send_keys("test_data_867")

    driver.quit()

if __name__ == "__main__":
    execute_test_47()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario47.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario47 {
    public static void executeTest47() {
        /*
         * Dummy test run for scenario 47
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_47");

        try {
            driver.findElement(By.cssSelector(".promo-code")).click();
            if (!driver.findElement(By.linkText("Log Out")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.id("user_name")).click();
            driver.findElement(By.xpath("//button[@type='submit']")).sendKeys("test_data_136");
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.linkText("Log Out")).sendKeys("test_data_210");
            Thread.sleep(2000);
            driver.findElement(By.linkText("Log Out")).sendKeys("test_data_759");
            driver.findElement(By.cssSelector(".promo-code")).click();
            driver.findElement(By.linkText("Log Out")).sendKeys("test_data_468");
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest47();
    }
}
